src/legacy/ui/public/vis/editors/default/default.js

```diff
@@ -1,200 +0,0 @@
-      this.vis = savedObj.vis;
-
-      if (!this.vis.type.editorConfig.optionTabs && this.vis.type.editorConfig.optionsTemplate) {
-        this.vis.type.editorConfig.optionTabs = [
-          {
-            name: 'options',
-            title: i18n.translate('common.ui.vis.editors.sidebar.tabs.optionsLabel', { defaultMessage: 'Options' }),
-            editor: this.vis.type.editorConfig.optionsTemplate,
-          }
-        ];
-      }
-    }
-
-    render({ uiState, timeRange, filters, query }) {
-      let $scope;
-
-      const updateScope = () => {
-        $scope.vis = this.vis;
-        $scope.uiState = uiState;
-        //$scope.$apply();
-      };
-
-      return new Promise(async (resolve) => {
-        if (!this.$scope) {
-          this.$scope = $scope = $rootScope.$new();
-
-          updateScope();
-
-          $scope.state = $scope.vis.copyCurrentState(true);
-          $scope.oldState = $scope.vis.getSerializableState($scope.state);
-
-          $scope.toggleSidebar = () => {
-            $scope.$broadcast('render');
-          };
-
-          this.el.one('renderComplete', resolve);
-          // track state of editable vis vs. "actual" vis
-          $scope.stageEditableVis = () => {
-            $scope.oldState = $scope.vis.getSerializableState($scope.state);
-            $scope.vis.setCurrentState($scope.state);
-            $scope.vis.updateState();
-            $scope.vis.dirty = false;
-          };
-          $scope.resetEditableVis = () => {
-            $scope.state = $scope.vis.copyCurrentState(true);
-            $scope.vis.dirty = false;
-          };
-
-          $scope.autoApplyEnabled = false;
-          if ($scope.vis.type.editorConfig.enableAutoApply) {
-            $scope.toggleAutoApply = () => {
-              $scope.autoApplyEnabled = !$scope.autoApplyEnabled;
-            };
-
-            $scope.$watch('vis.dirty', _.debounce(() => {
-              if (!$scope.autoApplyEnabled || !$scope.vis.dirty) return;
-              $scope.stageEditableVis();
-            }, 800));
-          }
-
-          $scope.submitEditorWithKeyboard = (event) => {
-            if (event.ctrlKey && event.keyCode === keyCodes.ENTER) {
-              event.preventDefault();
-              event.stopPropagation();
-              $scope.stageEditableVis();
-            }
-          };
-
-          $scope.getSidebarClass = () => {
-            if ($scope.vis.type.editorConfig.defaultSize === DefaultEditorSize.SMALL) {
-              return 'visEditor__collapsibleSidebar--small';
-            } else if ($scope.vis.type.editorConfig.defaultSize === DefaultEditorSize.MEDIUM) {
-              return 'visEditor__collapsibleSidebar--medium';
-            } else if ($scope.vis.type.editorConfig.defaultSize === DefaultEditorSize.LARGE) {
-              return 'visEditor__collapsibleSidebar--large';
-            }
-          };
-
-          $scope.$watch(() => {
-            return $scope.vis.getSerializableState($scope.state);
```

Comment:
If `getSerializableState` isn't used anymore, I believe we can remove it.

Reply:
Thanks, removed.